chain/jsonrpc/jsonrpc-tests/res/genesis_config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "protocol_version": 70,
+  "protocol_version": 71,
```

Comment:
We want to release BLS signatures as part of protocol version 70 release.

Reply:
fixed: https://github.com/near/nearcore/pull/11818/commits/ac29a2646c49bcdfbee9b8c335191e007a2e2ad3

https://github.com/near/nearcore/pull/11818/commits/2e39777b34acf4cf1b97a52b2b8c6ce292050b27